io-package.json

```diff
@@ -1,8 +1,11 @@
 {
   "common": {
     "name": "pvforecast",
-    "version": "2.3.0",
+    "version": "2.3.2",
     "news": {
+      "2.3.1": {
```

Comment:
News will be generated automatically (in all languages) by the release script (source is README.md).
Just remove this part. At the moment the version here doesn't match the io-package.json